src/concrete/ml/common/utils.py

```diff
@@ -38,7 +38,7 @@
 # when simulating FHE executions
 # Set 'USE_OLD_VL' to False by default once the new simulation is fixed
 # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4091
```

Comment:
also time to remove the fixme + comment above !

Reply:
Good catch thanks

Reply:
Done in #562